src/game_api/level_api.cpp

```diff
@@ -920,13 +920,13 @@ std::pair<int, int> LevelGenSystem::get_room_index(float x, float y)
         static_cast<int>(std::ceil(x - 3.5f)) / 10,
         static_cast<int>(std::ceil(121.5f - y)) / 8};
 }
-std::pair<float, float> LevelGenSystem::get_room_pos(int x, int y)
+std::pair<float, float> LevelGenSystem::get_room_pos(unsigned int x, unsigned int y)
```

Comment:
Please use uint32_t instead of unsigned int (we should also use int32_t imho but I think we're past that)